content/blog/configuring-azure-application-insights-in-an-angular-application/index.md

```diff
@@ -111,13 +116,41 @@ export class Insights {
         },
     });
 
-    // expose methods that can be used in components and services
-    trackEvent = this.appInsights.trackEvent;
-    trackTrace = this.appInsights.trackTrace;
-
     constructor(private router: Router, private appConfig: AppConfig) {
         this.appInsights.loadAppInsights();
     }
+
+    // expose methods that can be used in components and services
+    trackEvent(name: string): void {
+        this.appInsights.trackEvent({ name });
+    }
+
+    trackTrace(message: string): void {
+        this.appInsights.trackTrace({ message });
+    }
+}
+```
+
+## Initialize the `Insights` service
+
+The `Insights` service must be initialized before it can start logging views and exceptions. You can do that by injecting it in the `InsightsModule` module.
```

Comment:
```suggestion
The `Insights` service must be initialized before it can start logging views and exceptions. You can do that by injecting the service in the `InsightsModule` module. This way when the `InsightsModule` is loaded, the `Insights` service is also initialized.
```